docs/spec/spec.md

```diff
@@ -29,9 +29,9 @@ The conforming implementation of the specification is released and included in t
         2.3.2. [One-to-Many (1-n)](#232-one-to-many-1-n)  
 3. [Derived Entity Types and Persist Clients](#3-derived-entity-types-and-persist-clients)  
     3.1. [Derived Entity Types](#31-derived-entity-types)  
-    3.2. [Persist Clients](#32-persist-clients)
+    3.2. [Persist Clients](#32-persist-clients)  
 
-## 1. Overview  
+## 1. Overview
 
 The `persist` standard library creates a programming model to store and query data easily. This uses the statically typed nature of Ballerina to provide a convenient way to define and store the data model of an application across various providers such as RDBMS, NoSQL, and others. At this initial release `persist` library only supports MySQL data provider.
```

Comment:
```suggestion
The `persist` standard library creates a programming model to store and query data easily. This uses the statically typed nature of Ballerina to provide a convenient way to define and store the data model of an application across various providers such as RDBMS, NoSQL, and others. At this initial release, the `persist` library only supports the MySQL data provider.
```

Reply:
Fixed in [e4dc132](https://github.com/ballerina-platform/module-ballerina-persist/pull/109/commits/e4dc1324152adf53e52a1ca4dea90ffacb44ae7a)